.circleci/config.yml

```diff
@@ -758,6 +758,16 @@ jobs:
           version_path: versions/cwf_operator_version
       - magma_slack_notify
 
+  ###TeraVM Run_Status & Upgrade_Run_3GPP_Tests
+  Upgrade_Run_3GPP_Tests_teravm:
+    runner:
```

Comment:
there is an error on the check that says that `runner` is not a permitted option

https://app.circleci.com/pipelines/github/magma/magma/12854/workflows/62864c67-4b6e-43d5-93d6-3b0eeec1100f/jobs/113711